### DISCUSSION TITLE
Get items for the `Cooking Fest` 2024

### DISCUSSION THREAD

**Discussion by xTCry** [chr233/ASFEnhance#335] [杂谈 / General]
> [!IMPORTANT]  
> **Free — Valid only to _[18 November](https://steamcommunity.com/games/593110/announcements/detail/4464849834079158328)_**

### Use command

`RPI [Bots] 336074,336075,336076`

Use for all bots
```
RPI asf 336074,336075,336076
```

### Items

 - `336074` - *(Chat sticker)* **Simmering Pot** <img src="https://cdn.cloudflare.steamstatic.com/steamcommunity/public/images/items/3099220/11aaa8c0a629b45c191d158ce66f82c9ec8e2909.png" height="90" />
 - `336075` - *(Avatar frame)* **Bountiful Border** <img src="https://cdn.cloudflare.steamstatic.com/steamcommunity/public/images/items/3099220/1bdf31684c13fcd253273239e734acab3ca37149.png" height="90" />
 - `336076` - *(Animated avatar)* **Chopping Chef** <img src="https://cdn.cloudflare.steamstatic.com/steamcommunity/public/images/items/3099220/f8a3e4bcffd6a1b37bc3903e5ce2deaaf9938f27.gif" height="90" />

[View more on SteamDB (3099210)](https://steamdb.info/app/3099220/communityitems/)

**Comment by chr233** [comment]
thank you
